zulipterminal/ui_tools/views.py

```diff
@@ -806,6 +806,30 @@ def __init__(self, controller: Any,
         super().__init__(controller, widgets, 'GO_BACK')
 
 
+class AboutView(PopUpView):
+    def __init__(self, controller: Any, versions: Dict[str, str]) -> None:
+        content = [
+            ('Feature level', versions['feature_level']),
+            ('Server version', versions['server_version']),
+        ]
```

Comment:
This box seems like a great place to include the running version number of ZT? That might make it easier to split this commit into pieces - adding an about box, then adding various features to it?

Reply:
Yeah This is a nice idea! 👌

Reply:
That sounds plausible, thanks! :+1: